src/Behat/Behat/Context/Cli/InteractiveContextIdentifier.php

```diff
@@ -70,7 +70,7 @@ public function guessTargetContextClass(ContextEnvironment $environment)
 
         $message = $this->translator->trans('snippet_context_choice', array('%1%' => $suiteName), 'output');
         $choices = array_values(array_merge(array('None'), $contextClasses));
-        $default = current($contextClasses);
+        $default = key($contextClasses);
```

Comment:
This is probably not what we want here. It will always return `0` and point to `None`. This should potentially be a hard coded `1` as we already check if there's at least one context class above.